parser.yy

```diff
@@ -270,7 +270,7 @@ Expression:
 |	Expression '-' Expression
 |	Expression '.' Expression
 |	Expression '/' Expression
-|	Expression ':' Expression
+|	PATTERN ':' Term
```

Comment:
The intent of the original version was to allow Expression after ':' and later add a semantic check for the Expression before the ':' to be IDENT or PATTERN. This change does not need the semantic check, which I even forgot to add, but limits the thing after PATTER ':' to Term. It seems okay to me, but please confirm if you think it's correct.